executor_test.go

```diff
@@ -1505,8 +1507,20 @@ func TestExecutor_Execute_Row_Range(t *testing.T) {
 			}
 		})
 
+		t.Run("From", func(t *testing.T) {
```

Comment:
At first, I was a little surprised to see the implied `toTime` default to now + 1, but I suppose I understand the logic there. With that in mind, I guess my only comment is that there isn't actually a test which shows that a `Row()` query with only a `from` excludes results that are more than 1 day into the future. I guess if there was a record set at date `2200-01-01T00:00` then a test could ensure that it does not come back in the result set.

Reply:
I've updated the test to include a timestamp which is greater than the default end timestamp by 1 second to ensure that.

Reply:
I don't think that works on `f=10`; the query is checking `f=1`.
Also, I wonder if there's a case where `nextDayExclusive` ends up being the same as `toTime` since `time.Now()` is determined at different times. Perhaps add more than 1 second?

Reply:
Yeah, with 1 second it passed on local, but failed on CI. Added 1 day, waiting for the CI job to finish.

Reply:
The `nextDayExclusive.Format("2006-01-02T15:04")` is on the line with `f=10` but actually it affects the `Set(8, f=1, %s)` line. It seems to pass on the CI.

Reply:
Ah right. I see that now.